addons/web/static/src/js/views/basic/basic_renderer.js

```diff
@@ -656,7 +674,12 @@ var BasicRenderer = AbstractRenderer.extend({
             // wait for this.defs, we copy those classnames to the final element.
             widget.$el.addClass($el.attr('class'));
 
-            $el.replaceWith(widget.$el);
+            if (legacy) {
+                $el.replaceWith(widget.$el);
+            }
+            else {
+                widget.update(fieldWidgetProps);
```

Comment:
weird... will re-render the component

Reply:
I need to it because the component is not mounted.
If I do not do this then the component is unmounted and we could not do anything :/
I spoke with @kebeclibre about this